src/clue/app-config.json

```
@@ -10,6 +10,8 @@
   "defaultDocumentType": "problem",
   "defaultDocumentTitle": "Untitled",
   "defaultLearningLogTitle": "UntitledLog",
+  "defaultInitialLearningLogTitle": "My First Learning Log",
```

```suggestion
  "initialLearningLogTitle": "My First Learning Log",
```
It's the name of the initial learning log, not a default in the same sense the others are.

---

I had assumed it'd be considered a default but that would also make sense as to why it wouldn't be considered of the same nature as the other [more default] defaults.